sdk/schemaregistry/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Schema Registry client library for JavaScript
+
+We have two client libraries for Azure Schema Registry
+
+- `@azure/schema-registry` - The Schema Registry client for storing and retrieving schemas. You can find more info over [here](./schema-registry).
+- `@azure/schema-registry-avro` - The Avro-based serializer that leverages Schema Registry. You can find more info over [here](./schema-registry-avro).
+
+## Getting started
```

Comment:
I think we should consider adding a README exception for this file since having this placeholder content feels silly. The file you need to add the omission to is here: https://github.com/Azure/azure-sdk-for-js/blob/master/eng/.docsettings.yml